class TaskNotFoundException(Exception):
    pass


class TaskNotCompletedException(Exception):
    pass
